src/server.js

```diff
@@ -669,7 +669,7 @@ function initializeClient(socket, client, token, lastMessage, openChannel) {
 				chan.setMuteStatus(setMutedTo);
 			}
 
-			socket.emit("mute:changed", {
+			manager.sockets.emit("mute:changed", {
```

Comment:
This seems wrong.
You aren't sending the mute change event to the User are you?
Rather you send it to every connected user (manager.sockets is the global io object)

Or do I misunderstand how our transport layer works?

Reply:
Good catch, I rushed. Proper method is to use the socket IDs and find the relevant one in manager.sockets.